tests/sanity/test/install/test-installed-files.js

```diff
@@ -10,6 +10,12 @@
 
 const sshHelper = require('./ssh-helper');
 
+const chai = require('chai');
+const chaiFiles = require('chai-files');
```

Comment:
I search `chai-files` in the Changes and only find this line, did you miss defining this as `devDependencies`?

Reply:
Now it looks like this
```
const sshHelper = require('./ssh-helper');
const expect = require('chai').expect;
const debug = require('debug')('zowe-sanity-test:install:installed-files');
const addContext = require('mochawesome/addContext'); 
```